.github/workflows/ci.yml

```diff
@@ -1,103 +1,199 @@
-
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v2
-
-      - name: Fix Template
-        uses: hashicorp/packer-github-actions@master
-        with:
-          command: fix
-          target: "packer_templates/${{ matrix.template }}"
-
-      - name: Validate Template
-        uses: hashicorp/packer-github-actions@master
-        with:
-          command: validate
-          arguments: -syntax-only
-          target: "packer_templates/${{ matrix.template }}"
-
-  enumerate_iso_validation:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: enumerate_iso_validation
-        env:
-          EXCLUDE: 'amazon-2*,rhel*,sles*,solaris*'
-        shell: pwsh
-        run: "'::set-output name=matrix::{0}' -f $(Get-ChildItem -Path packer_templates -Include *.json -Exclude ${{ env.EXCLUDE }} -Recurse | Foreach-Object {Join-Path -Path $_.Directory.Name -ChildPath $_.Name} | ConvertTo-Json -Compress)"
-        id: enumerate_iso_validation
-
-    outputs:
-      matrix: ${{ steps.enumerate_iso_validation.outputs.matrix }}
-
-  iso-availability:
-    needs: enumerate_iso_validation
-
-    runs-on: ubuntu-latest
-    name: iso-validate
-
-    strategy:
-      fail-fast: false
-      matrix:
-        template: ${{ fromJson(needs.enumerate_iso_validation.outputs.matrix) }}
-
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v2
-      - name: iso-availability
-        shell: pwsh
-        run: |
-          $(Get-Content "packer_templates/${{ matrix.template }}" | ConvertFrom-Json ) | 
-            Foreach-Object {
-              if($_.variables.iso_url){
-                $_.variables.iso_url
-              }else{
-                '{0}/{1}/{2}' -f $_.variables.mirror,$_.variables.mirror_directory,$_.variables.iso_name
-              }
-            } | Foreach-Object {
-              Invoke-RestMethod -Uri $_ -Method Head -Verbose
-            }
+---
+on:
+  pull_request:
+
+jobs:
+  markdownlint:
+    uses: Stromweld/github-workflows/.github/workflows/markdownlint.yml@main
+
+  md-links:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@main
+      - name: Pull Change list
+        id: changed-files
+        uses: collin-miller/git-changesets@master
+      - name: Verify Changed files
+        id: verify-changed-files
+        run: |
+          echo '${{ steps.changed-files.outputs.added_modified }}' > changed.txt
+          if grep ".*\.md" changed.txt; then
+            echo "::set-output name=files_changed::true"
+          else
+            echo "::set-output name=files_changed::false"
+          fi
+      - name: markdown-link-check
+        if: steps.verify-changed-files.outputs.files_changed == 'true'
+        uses: gaurav-nelson/github-action-markdown-link-check@master
+        with:
+          use-verbose-mode: "yes"
+
+  yamllint:
+    uses: Stromweld/github-workflows/.github/workflows/yamllint.yml@main
```

Comment:
Was this source intentional?

Reply:
yes for now till I can get with the pipeline guys and find an official chef home them.